packages/components/src/FormField/hooks/useFormFieldFocus.ts

```diff
@@ -56,7 +49,7 @@ export function useFormFieldFocus(
         handleFocusOut,
       );
     };
-  }, [props.wrapperRef?.current]);
```

Comment:
this wasn't doing anything since ref updates don't cause re-renders. we should look at getting the exhaustive deps linting rule in here. I believe it would have flagged this.

Reply:
seems like `exhaustive-deps` rule would be relatively simple to add.  Maybe something to put side desk or write a ticket for